packages/dashboard-core-plugins/src/ChartPlugin.tsx

```diff
@@ -63,8 +69,7 @@ export const ChartPlugin = (props: ChartPluginProps): JSX.Element => {
     };
   }, [hydrate, registerComponent]);
 
-  useListener(layout.eventHub, ChartEvent.OPEN, handleOpen);
-  useListener(layout.eventHub, ChartEvent.CLOSE, handleClose);
+  useListener(layout.eventHub, PanelEvent.OPEN, handlePanelOpen);
```

Comment:
Looks like `IrisGridPanel` still emits `ChartEvent.OPEN` in `handleCreateChart`

Reply:
I've created a new `IrisGridEvent.CREATE_CHART` to remove the dependency of grid onto chart. In the future, we should think of passing Chart Builder functionality in to the Iris Grid Menu so it can be defined completely externally.